src/libraries/Microsoft.PowerFx.Core/Errors/ErrorSeverity.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx
+{
+    /// <summary>
+    /// Severity of errors provided.
+    /// </summary>
+    public enum ErrorSeverity
+    {
+        // These values are a mirror of Microsoft.PowerFx.Core.Errors.DocumentErrorSeverity, 
+        // which is an internal type with [TransportType] attribute.
+
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+        _Min = Verbose,
+#pragma warning restore SA1300 // Element should begin with upper-case letter
+
+        /// <summary>
+        /// A suggestion about possible high-level improvements or refactoring that may help the user
+        /// get a better app experience.
+        /// Examples: performance changes
+        /// Verbose messages will not be part of 'ChoreUpdateRulesWithErrors' as they would be analysed when the dependency changes.
```

Comment:
Nits: Chore references don't belong in public facing doc comments (apply to only PAClient analysis)

Reply:
removed.